contracts/upgradeable/MultiCallerUpgradeable.sol

```diff
@@ -0,0 +1,35 @@
+// SPDX-License-Identifier: AGPL-3.0-only
+pragma solidity ^0.8.0;
+
+/**
+ * @title MockSpokePool
+ * @notice Logic is 100% copied from "@uma/core/contracts/common/implementation/MultiCaller.sol" but one
```

Comment:
We should remember to exclude this contract from the audit scope considering the contract's logic has already been audited.